locator/token_metadata.cc

```diff
@@ -361,6 +366,7 @@ future<token_metadata_impl> token_metadata_impl::clone_only_token_map(bool clone
         return do_for_each(_token_to_endpoint_map, [&ret] (const auto& p) {
             ret._token_to_endpoint_map.emplace(p);
         }).then([this, &ret] {
+            ret._nr_normal_token_owners = _nr_normal_token_owners;
             ret._endpoint_to_host_id_map = _endpoint_to_host_id_map;
         }).then([this, &ret] {
```

Comment:
while at it, why do we need 3 `.then` continuations here where each of them is synchronous?
Even if the compiler is able is optimize them as unconditionally ready, why waste those compiler cycles?

Reply:
Git blame says you added some of the code ;) 

E.g., 

4fc5997949c0a4bc42ce37af617e75daa9f0af57 for token_metadata_impl::clone_async

Reply:
Yeah, I followed the "convention".
Should have done that then